packages/fxa-settings/src/pages/ResetPassword/container.tsx

```diff
@@ -0,0 +1,177 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { gql, useMutation } from '@apollo/client';
+import ResetPassword from '.';
+import { MozServices } from '../../lib/types';
+import { useCallback, useState } from 'react';
+import {
+  AuthUiErrorNos,
+  AuthUiErrors,
+  composeAuthUiErrorTranslationId,
+} from '../../lib/auth-errors/auth-errors';
+import { RouteComponentProps } from '@reach/router';
+import { GraphQLError } from 'graphql';
+import { useValidatedQueryParams } from '../../lib/hooks/useValidate';
+import {
+  AccountRecoveryKeyInfo,
+  ResetPasswordQueryParams,
+} from '../../models/reset-password';
+import LoadingSpinner from 'fxa-react/components/LoadingSpinner';
+
+// WIP PROTOTYPE
+// ❌ = Note for prototype review. Comment will be removed in implementation.
+
+export interface PasswordForgotSendCodeResponse {
+  passwordForgotSendCode: {
+    passwordForgotToken: string;
+  };
+}
+
+export type BeginResetPasswordHandler = (
```

Comment:
A `type` sandwiched between two `interface`s so I'm curious why that is.

Reply:
Heh, I used to use `type` whenever I could and only used `interface` when it was needed (for extending). Recently Julian mentioned we should just use `interface` everywhere when possible, and I don't have a strong preference so I'm pretty sure I just used it out of habit here. 😅

Reply:
👍 

fwiw for function types I prefer `type`.  It may be superficial but I find `=>` for the return type easier to see/read than `:`.